go-msgpack/codec/rpc.go

```diff
@@ -6,8 +6,9 @@ package codec
 import (
 	"bufio"
 	"io"
-	"net/rpc"
 	"sync"
+
+	"github.com/hashicorp/consul-net-rpc/net/rpc"
```

Comment:
lines 138 and 141 in this file have comment mentions to net/rpc.

Not sure if we'd want to add guidance on the fact that our fork is used from now on.

just calling this out 🤷🏼

Reply:
Good call out! I don't have a strong preference either way. I doubt anyone reading this fork would be confused by it as it's still the relative package name of this package.